### PR TITLE
Changes from Dan

### PR DESCRIPTION
**Pull request by chendaniely**
this is a new file created to show forking and pull requests

### CODE REVIEW

**Comment by chendaniely** [comment]
@dbouquin will you accept this pull request? :+1:

**Comment by dbouquin** [comment]
Sure!

**Comment by dbouquin** [comment]
So I'll click "merge pull request"

File: dan_changes.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Hello!
+Thanks for hosting a github workshop!
+
```
Comment (chendaniely) [inline dan_changes.txt:3]: Should I leave this last empty line here?

Comment (dbouquin) [inline dan_changes.txt:3]: Yeah that line is awesome

**Pull request merged**